### PR TITLE
CRM-19783 - Added form input labels for mailing list groups and remov…

### PR DESCRIPTION
**Pull request by triprems**
Accessibility updates for Subscribe.tpl

### CODE REVIEW

File: templates/CRM/Mailing/Form/Subscribe.tpl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {*
  +--------------------------------------------------------------------+
- | CiviCRM version 4.7                                                |
+ | CiviCRM version 4.5                                                |
```
Comment (yashodha) [inline templates/CRM/Mailing/Form/Subscribe.tpl]: @triprems : Can you make sure you only make the needful changes? It seems like you were using an old version of file to make changes.

File: templates/CRM/Mailing/Form/Subscribe.tpl [hunk 2]
```diff
@@ -26,11 +26,11 @@
 {* this template is used for web-based subscriptions to mailing list type groups  *}
 <div class="crm-block crm-mailing-subscribe-form-block">
 {if $single}
-    <div class="help">
+    <div id="help">
```
Comment (yashodha) [inline templates/CRM/Mailing/Form/Subscribe.tpl]: It seems like you are undoing some of the changes as you are using earlier version of this file.

File: templates/CRM/Mailing/Form/Subscribe.tpl [hunk 3]
```diff
@@ -26,11 +26,11 @@
...
         {ts}Enter your email address and click <strong>Subscribe</strong>. You will receive a confirmation request via email shortly. Your subscription will be activated after you respond to that email.{/ts}
     </div>
 {else}
-    <div class="help">
+    <div id="help">
```
Comment (yashodha) [inline templates/CRM/Mailing/Form/Subscribe.tpl]: Ditto

**Comment by triprems** [comment]
Thanks, I'll get the latest file, and will re-submit this pull request after the holidays.

**Comment by yashodha** [comment]
alternate PR for https://github.com/civicrm/civicrm-core/pull/9606

**Pull request closed without merging**